src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/MarkdownHelper.java

```diff
@@ -23,7 +23,16 @@ public static String createTwoColumnTable(String[] header, List<String[]> rows)
             throw new IllegalArgumentException("header array should contain two items");
```

Comment:
What is header is greater than 2?

Reply:
To make a two column table, the String array passed in must contain two items